src/Roslyn.Suppressions/Roslyn.Suppressions/SettingsFile/RoslynSettingsFileInfo.cs

```diff
@@ -28,12 +28,13 @@ internal static class RoslynSettingsFileInfo
         public static readonly string Directory = PathHelper.GetTempDirForTask(false, "Roslyn");
 
         /// <summary>
-        /// Returns the full file path for the given project key
+        /// Returns the full file path for the given solution name
         /// </summary>
+        /// <param name="solutionName">The solution name without the file extension e.g. "MySolution"</param>
         /// <remarks> File will be in a shared location and would be able to be accessed by multiple instances of VS simulateneously</remarks>
-        public static string GetSettingsFilePath(string sonarProjectKey)
+        public static string GetSettingsFilePath(string solutionName)
         {
-            var escapedName = PathHelper.EscapeFileName(NormalizeKey(sonarProjectKey));
+            var escapedName = PathHelper.EscapeFileName(NormalizeKey(solutionName));
```

Comment:
Strictly speaking it shouldn't be necessary to call `PathHelper.EscapeFileName` since we are expecting to be passed in a file name.
However, it was an unnecessary change at this point so I left it as-is.